cfgov/unprocessed/css/molecules/multi-select.less

```diff
@@ -110,12 +109,14 @@
                 }
             }
 
-            &:after {
+            & span:after {
                 .cf-icon();
+                display: inline;
```

Comment:
Is this necessary since you're absolutely positioning it?

Reply:
Weird, I don't know why GH is hiding this comment, it's not outdated and still stands.